io/disk/ssd/nvmetest.py

```diff
@@ -196,8 +196,9 @@ def get_ns_controller(self):
         cmd = "%s list-ctrl %s" % (self.binary, self.device)
         output = process.system_output(cmd, shell=True,
                                        ignore_status=True).decode("utf-8")
-        if output:
-            return output.split(':')[-1]
+        for line in output.splitlines():
+            if '0]' in line:
```

Comment:
Will `output.splitlines()[0].split(':')[-1]` always work?

Reply:
No, we need controller ID from the line containing `[   0]`